src/Controls/tests/UITests/Tests/Issues/Issue21513.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue21513 : _IssuesUITest
+	{
+		public Issue21513(TestDevice device)
+			: base(device)
+		{ }
+
+		public override string Issue => "Buttons with images don't cover text";
+
+		[Test]
+		public void Issue21513Test()
+		{
+			this.IgnoreIfPlatforms(new [] { TestDevice.Mac, TestDevice.iOS, TestDevice.Android });
```

Comment:
Can you also run this on android and generate the images there? Hopefully they are the same and we don't have to fix anything on android